test/testHelpers.js

```diff
@@ -52,3 +52,8 @@ export function destroyFullToggle(redactor, toggleIndex, blocks) {
     currentBlock.remove();
   }
 }
+
+export function extractionBlock(redactor, toggleIndex) {
+  const currentBlock = redactor.children[toggleIndex];
+  currentBlock.remove();
```

Comment:
@Jenietoc when a block is extracted from the toggle it should be sent after the toggle (root + children). To simulate the original behavior, you can do something like the following. Remember that the number of blocks must be preserved.


```js
export function createDefaultBlock(data) {
  const newBlock = document.createElement('div');
  newBlock.classList.add('ce-block');

  const content = document.createElement('div');
  content.classList.add('ce-block__content');

  const paragraph = document.createElement('div');
  paragraph.classList.add('ce-paragraph', 'cdx-block');
  paragraph.contentEditable = true;
  paragraph.textContent = data.text;

  content.appendChild(paragraph);
  newBlock.appendChild(content);

  return newBlock;
}

export function extractionBlock(toggleBlock, redactor, toggleIndex) {
  const currentBlock = redactor.children[toggleIndex];
  const { items } = toggleBlock.data;
  const { data } = items[toggleIndex - 1];
  const extractedBlock = createDefaultBlock(data);

  currentBlock.remove();
  redactor.appendChild(extractedBlock);
}
```


**In the main test file:**
```js
    it('Extract de first block into the toggle', () => {
      const children = document.querySelectorAll(`div[foreignKey="${toggleBlock.wrapper.id}"]`);
      extractionBlock(toggleBlock, redactor, 1);

      expect(redactor.children.length).toBe(4);
      expect(children.length).toBe(3);
    });
```